server/policy_constants.go

```diff
@@ -18,13 +18,18 @@ import (
 const (
 	NoAction                   uint32 = 0x00000003
 	Separator                  uint32 = 0x00000004
+	EventTag                   uint32 = 0x00000006
 	SCRTMVersion               uint32 = 0x00000008
 	IPL                        uint32 = 0x0000000D
 	NonhostInfo                uint32 = 0x00000011
 	EFIBootServicesApplication uint32 = 0x80000003
 	EFIAction                  uint32 = 0x80000007
 )
 
+// EventTagHex used with type "EV_EVENT_TAG".
+// This corresponds to a TLV struct of type LOAD_OPTIONS_EVENT_TAG_ID (0x8F3B22ED, reversed endian), length 0x1a (26), value `LOADED_IMAGE::LoadOptions\n`.
+const EventTagHex = "ed223b8f1a0000004c4f414445445f494d4147453a3a4c6f61644f7074696f6e7300"
```

Comment:
nit: EventTagLoadedImageHex to be more specific

Reply:
Done.